modules/ROOT/pages/auradb/managing-databases/advanced-metrics.adoc

```diff
@@ -1,6 +1,7 @@
 [[aura-monitoring]]
 = Advanced metrics
 
+label:AuraDB-Professional[]
 label:AuraDB-Enterprise[]
```

Comment:
Will be available for AuraDB Pro and Enterprise (but not free!). Certain features limited to Enterprise, as indicated by labels in the following text.